packages/common/test/ArcGISContextManager.test.ts

```diff
@@ -20,6 +20,25 @@ import { IPortal } from "@esri/arcgis-rest-portal";
 import ExceptionFactory from "./mocks/ExceptionFactory";
 import { TOMORROW } from "./test-helpers/tomorrow";
 
+const onlineUserResponse = {
```

Comment:
had to re-order things so I could clone this into the objects defined lower down